plugins/generator-1.21.1/neoforge-1.21.1/templates/livingentity/livingentity_renderer.java.ftl

```diff
@@ -153,6 +153,9 @@ public class ${name}Renderer extends <#if humanoid>Humanoid</#if>MobRenderer<${n
 		<#if data.mobModelName == "Villager">
 			poseStack.scale(0.9375f, 0.9375f, 0.9375f);
 		</#if>
+		<#if data.breedable>
+			poseStack.scale(entity.getAgeScale(), enity.getAgeScale(), entity.getAgeScale());

```

Comment:
This is why tests are failing.

```suggestion
			poseStack.scale(entity.getAgeScale(), entity.getAgeScale(), entity.getAgeScale());

```

Reply:
Ah, forgot to fix in 1.21.1 also. Thanks!